3.0.13 Build fails with libcap 2.29+

```
lib/util.c:582:12: error: conflicting types for 'cap_setuid'
  582 | static int cap_setuid(int uid, int is_master)
      |            ^~~~~~~~~~
In file included from lib/util.c:57:
/usr/include/sys/capability.h:138:12: note: previous declaration of 'cap_setuid' was here
  138 | extern int cap_setuid(uid_t uid);
      |            ^~~~~~~~~~
make[2]: *** [Makefile:5747: lib/lib_libcyrus_min_la-util.lo] Error 1
```

https://github.com/cyrusimap/cyrus-imapd/blob/95bfb40c041704158be48758d8ab3782caf1f980/lib/util.c#L582

--- a/lib/util.c
+++ b/lib/util.c
@@ -666,7 +666,7 @@ EXPORTED int set_caps(int stage __attribute__((unused)),
 }
 #endif
 
-static int cap_setuid(int uid, int is_master)
+static int cyrus_cap_setuid(int uid, int is_master)
 {
     int r;
 
@@ -686,7 +686,7 @@ EXPORTED int become_cyrus(int is_master)
     int result;
     static uid_t uid = 0;
 
-    if (uid) return cap_setuid(uid, is_master);
+    if (uid) return cyrus_cap_setuid(uid, is_master);
 
     const char *cyrus = cyrus_user();
     const char *mail = cyrus_group();
@@ -732,7 +732,7 @@ EXPORTED int become_cyrus(int is_master)
         return -1;
     }
 
-    result = cap_setuid(newuid, is_master);
+    result = cyrus_cap_setuid(newuid, is_master);
 
     /* Only set static uid if successful, else future calls won't reset gid */
     if (result == 0)

--- a/master/master.c
+++ b/master/master.c
@@ -297,7 +297,7 @@ static void get_statsock(int filedes[2])
         fatalf(1, "unable to set close-on-exec: %m");
 }
 
-static int cap_bind(int socket, struct sockaddr *addr, socklen_t length)
+static int cyrus_cap_bind(int socket, struct sockaddr *addr, socklen_t length)
 {
     int r;
 
@@ -663,7 +663,7 @@ static void service_create(struct service *s, int is_startup)
 #endif
 
         oldumask = umask((mode_t) 0); /* for linux */
-        r = cap_bind(s->socket, res->ai_addr, res->ai_addrlen);
+        r = cyrus_cap_bind(s->socket, res->ai_addr, res->ai_addrlen);
         umask(oldumask);
         if (r < 0) {
             int e = errno;

